packages/vuepress-theme-odyssey/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@okta/vuepress-theme-odyssey",
+  "version": "0.6.0",
+  "description": "VuePress theme for Odyssey Documentation",
+  "author": "Okta, Inc.",
+  "license": "Apache-2.0",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint . --ext .js,.vue && stylelint ./*.scss",
```

Comment:
Let's move this to the root in favor of running ESLint on all files and folders. By default, ESLint will extend the root config with package configuration file, like you have [here](https://github.com/okta/odyssey/blob/3ed88d51d674f76f36ca172fc4f31eb8b8c22f4c/packages/vuepress-theme-odyssey/.eslintrc.js)

Reply:
Moved comment to #729